pages/bids.py

```diff
@@ -0,0 +1,19 @@
+import os
+import pandas as pd
+import streamlit as st
+
+def main():
+    st.title("All non-BIDS compliant files on lincbrain.org")
+
+    AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+    AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+
+    df = pd.read_csv(
+        f"s3://linc-dashboard/lincbrain_assets.csv",
+        storage_options={
+            "key": AWS_ACCESS_KEY_ID,
+            "secret": AWS_SECRET_ACCESS_KEY,
+        },
+    )
```

Comment:
You call this code block in all three `pages/` files -- you could create a `utils.py` and import it if that would be useful

Reply:
Good idea.  Just pushed a commit based on this suggestion.